test/integration/testdata/yaml/activestate.yaml

```diff
@@ -0,0 +1,89 @@
+project: https://platform.activestate.com/ActiveState-CLI/Yaml-Test/?branch=main&commitID=0476ac66-007c-4da7-8922-d6ea9b284fae
```

Comment:
Rather than have this hard coded on the filesystem it'd be nice if we can generate this on the fly from the test. That way the code for the actual test we're performing is all in one place also.

So the individual tests would be doing something like:

```
suite.testScriptPerformance(`echo "Hello World"`, "Hello World", DefaultSamples, baseline)
```

Reply:
It proved to be a bit more complex when we want to support tests calling other scripts, using constants and calling scripts from merged activestate.yaml files.